openshift/e2e-common.sh

```diff
@@ -193,10 +195,10 @@ EOF
   value:
     name: "kourier-manifest"
     configMap:
-      name: "kourier-cm"
+      name: "empty-cm"
       items:
-        - key: "kourier.yaml"
-          path: "kourier.yaml"
+        - key: "empty.yaml"
+          path: "empty.yaml"
```

Comment:
Could we also use an `emptyDir` mount and not create the empty configmap at all?

Reply:
Yes! It should be possible. Thank you!